server.coffee

```diff
@@ -24,5 +24,11 @@ application = module.exports = (callback) ->
         app.use errorMiddleware
         initialize app, server, callback
 
+        axon = require 'axon'
+        socket = axon.socket 'sub-emitter'
+        socket.connect 9105
+        CozyInstance = require './server/models/instance'
+        socket.on 'cozyinstance.update', CozyInstance.updateDomainCache
```

Comment:
Are you sure you need that if you use the Cozy Realtime Adapter? It looks like it already does that: https://github.com/cozy/cozy-realtime-adapter#server-side--axon---socketio